spec/services/copy_course_from_production_spec.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe CopyCourseFromProduction do
+  let(:existent_prod_course) { 'https://dashboard.wikiedu.org/courses/University_of_California_Merced/Feminism,_Handmaids_and_Wild_Seeds_(Spring_2021)' }
+
+  context 'with courses in production' do
+    let(:subject) do
+      described_class.new(existent_prod_course)
+    end
+
+    it 'copy course to dev env' do
```

Comment:
This should test more than just the returned `result`. It should verify that examples of each of the important types of data got copied over as well: students, instructors, and assignments.

Reply:
Do you think this kind of examples are enough? [6e16b75](https://github.com/WikiEducationFoundation/WikiEduDashboard/pull/4914/commits/6e16b75e308a948ef00753b658878b351947d7b6)

Reply:
Looks good.